src/app/feed/feed-advanced-search/feed-advanced-search.component.spec.ts

```diff
@@ -30,7 +30,156 @@ describe('Component: FeedAdvancedSearchComponent', () => {
 	});
 
 	it('should create an instance', async(() => {
-		const component = TestBed.createComponent(FeedAdvancedSearchComponent);
+		const fixture = TestBed.createComponent(FeedAdvancedSearchComponent);
+		const component = fixture.debugElement.componentInstance;
 		expect(component).toBeTruthy();
 	}));
+
+	it('should have an toolList', async(() => {
+		const fixture = TestBed.createComponent(FeedAdvancedSearchComponent);
+		const compiled = fixture.debugElement.nativeElement;
+
+		expect(compiled.querySelector('.wrapper .tools .tool-list')).toBeTruthy();
+	}));
+
+	it('should set toolList on ngOnInit() and blank query', async(() => {
+		const fixture = TestBed.createComponent(FeedAdvancedSearchComponent);
+		const component = fixture.debugElement.componentInstance;
+		component.ngOnInit();
+		expect(component.toolList.inert).toBe(true);
+		const query$ = component.store.select(fromRoot.getQuery);
+		let displayString: string;
+		const subscription = query$.subscribe(query => displayString = query.displayString);
+		expect(displayString).toBe('');
+		subscription.unsubscribe();
```

Comment:
Is it explicitly required?

Reply:
@simsausaurabh  Actually in this block of code, I am just testing that `getDataFromStore()` of `feed-advanced-search.component.ts` is returning empty string to `query$` initially.

Reply:
@simsausaurabh please review this